### ISSUE TITLE
function autocompletion inside module not working

### ISSUE THREAD

**Issue by nulian** [KronicDeth/intellij-elixir#1254]
Should it work with the plugin at the moment autocompletion of functions that you have inside the module. Function auto completion in other modules work and variable autocompletion. But i'm not getting it to work inside the same file when you don't prefix it with a module name.

Tried it with latest plugin and intellij ce edition. Switching over from rubymine because of better way to include deps folder as a library.

**Comment by KronicDeth** [comment]
While not the same as #1251, this had the same root cause and so has been fixed by #1256.

**Comment by nulian** [comment]
This 9.0 version doesn't seem to have fixed it yet.
But I found what is causing it. When you have a module inside a module autocompletion of functions breaks.
Probably because the local method scope switches to the module inside it.

```
defmodule Autocomplete do

  defmodule State do
    def another_test do
    end
  end

def test do
  # you will see `test` here in the list and `another_test`
 # But you will not see `internal_test`
end

defp internal_test do
end

end

```
